dev/com.ibm.ws.microprofile.faulttolerance.3.0.metrics/src/com/ibm/ws/microprofile/faulttolerance/metrics/integration/package-info.java

```diff
@@ -0,0 +1,18 @@
+/*******************************************************************************
+ * Copyright (c) 2018 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+/**
+ * @version 1.0
+ */
+@org.osgi.annotation.versioning.Version("1.0")
+@TraceOptions(traceGroup = "FAULTTOLERANCE")
+package com.ibm.ws.microprofile.faulttolerance.metrics.integration;
```

Comment:
Should this package be `com.ibm.ws.microprofile.faulttolerance30.metrics.integration` to avoid an future clashes?

Reply:
yes